### ISSUE TITLE
how to suppress warning with mpl_toolkits when matplotlib installed

### ISSUE THREAD

**Issue by lisphilar** [fpgmaas/deptry#34]
`mpl_toolkits` is a part of `matplotlib` and we can ignore the following warning when `matplotlib` is included in pyproject.toml file.

```Python
Warning: Failed to find corresponding package name for import `mpl_toolkits` in current environment.
```

No priority.

**Comment by fpgmaas** [comment]
Thanks @lisphilar for raising this issue. I see that there is a file called `top_levels.txt` in the metadata of `matplotlib`. It should be possible to include this information during the checking for obsolete dependencies.

This will also help with #5, since these packages also have a file called `top_levels.txt`, stating `google` as the top level.

**Comment by fpgmaas** [comment]
@lisphilar Should now be fixed in version 0.2.2!

**Comment by lisphilar** [comment]
@fpgmaas , Thank you always for your quick release!!

Finally I confirmed that version 0.2.2 (0.2.3) removed the warning.
Best Regards
